### PR TITLE
Draft: Add logistics

### PR DESCRIPTION
**Pull request by ibzm**
Controller: Added show() method to display individual order details, and fetched items using the relationship.
Route: Defined logistics.show route to handle order details.
Models: Defined items() relationship in the Order model to retrieve related items.

Views:
`logistics.blade.php:`  Displays list of orders.
`show.blade.php:`  Displays detailed order information, including items, and includes a back button to return to the order list.

This structure ensures that users can navigate through the logistics orders and view the detailed items in each order with easy navigation back to the main logistics page.

### CODE REVIEW

**Comment by WebExtension1** [comment]
- Add "fulfilled" boolean to the "order" table. Either set it's default value to 0, or insert it as 0 on every new order.

- Suggested: Append the "delivery_note" table to be a "delivery" table, with "delivery_note" being an id you can enter on the page which isn't unique.

- Make this data actually be inputted into the database, including a check to see if the order has been fulfilled, changing the "fulfilled" status in the "order" table if the status has changed.

- Consult AJ on styling.

File: resources/views/Logistics/index.blade.php [hunk 1]
```diff
@@ -1,7 +1,50 @@
...
-            {{ __('Logistics') }}
+            {{ __('Logistics Dashboard') }}
         </h2>
     </x-slot>
+
+    <div class="bg-stockhive-grey-dark text-white shadow-sm rounded-lg mt-8 lg:w-[85%] w-full m-auto p-4">
+        <div class="flex justify-between items-center gap-8 my-4 border-grey bg-stockhive-grey rounded-lg p-4 border-2 m-auto w-[90%] text-right">
+            <h1 class="text-lg font-semibold">Logistics Dashboard</h1>
+            </a>
+        </div>
```
Comment (ajstrongdev) [inline resources/views/Logistics/index.blade.php:12]: Remove this title it's duplicated and doesn't follow the rest of the site theme.

File: resources/views/Logistics/index.blade.php [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+                    @foreach ($orders as $order)
+                        <tr>
+                            <td class="py-2 px-4">{{ $order->id }}</td>
+                            <td class="py-2 px-4">{{ $order->user_id }}</td>
+                            <td class="py-2 px-4">{{ $order->store_id }}</td>
+                            <td class="py-2 px-4">{{ $order->date_time }}</td>
+                            <td class="py-2 px-4">
+                            <x-primary-button>View Order</x-primary-button>
+                            <td class="py-2 px-1">
+                            <td class="py-2 px-4">
```
Comment (ajstrongdev) [inline resources/views/Logistics/index.blade.php:35]: Unclosed/Unused `<td>` tag. Fix this.

File: resources/views/Logistics/index.blade.php [hunk 3]
```diff
@@ -1,7 +1,50 @@
...
+                                
+      <a href="{{ route('logistics.show', $order->id) }}">
+        <x-primary-button>Create Delivery</x-primary-button>
+        </a>
```
Comment (ajstrongdev) [inline resources/views/Logistics/index.blade.php:39]: Why is this in an `<a>` tag. Routes exist to protect get/post requests, you **should/must** be routing from within a form.

**Review by ajstrongdev — CHANGES_REQUESTED** [review]

**Pull request merged**